manifests/0000_26_cloud-controller-manager-operator_00_namespace.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    openshift.io/node-selector: ""
```

Comment:
What's this annotation for? 🤔

Reply:
This is a plain merge/copy from https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/manifests/0000_25_kube-controller-manager-operator_00_namespace.yaml

Reply:
I believe it comes from OpenShift 3: https://docs.openshift.com/container-platform/3.11/admin_guide/managing_projects.html#setting-the-project-wide-node-selector

> If openshift.io/node-selector is set to an empty string (oc adm new-project --node-selector=""), the project will not have an administrator-set node selector, even if the cluster-wide default has been set. This means that, as a cluster administrator, you can set a default to restrict developer projects to a subset of nodes and still enable infrastructure or other projects to schedule the entire cluster.

So basically this annotation unsets the default cluster-wide node-selector for the namespace. I don't know if it's still valid for OpenShift 4 but I suggest to keep the annotation.